framework/components/blockchain/containers.go

```diff
@@ -13,29 +13,32 @@ import (
 	"github.com/smartcontractkit/chainlink-testing-framework/framework"
 )
 
-func baseRequest(in *Input) testcontainers.ContainerRequest {
+func baseRequest(in *Input, useWS bool) testcontainers.ContainerRequest {
```

Comment:
wdyt about this tiny change?
```
type ExposeWs = bool
const (
       WithWsEndpoint ExposeWs = true
       WithoutWsEndpoint ExposeWs = false
)

func baseRequest(in *Input, useWS ExposeWs) testcontainers.ContainerRequest {
```

that would eliminate the need to open the functions definition in order to understand what `true/false` means.